azure-pipelines/official.yml

```diff
@@ -4,6 +4,18 @@ trigger:
   paths:
     exclude: ["*.md"]
 
+schedules:
+  - cron: "0 8 23-29 * 0"
+    displayName: "Monthly build check"
+    branches:
+      include: 
+        - main
+        - rel/*
+      exclude: 
+        - ""
```

Comment:
What does this accomplish?

Reply:
Denotes the location for the exclude, should we ever need one.  Will remove